fiddler/requirements-dev.txt

```diff
@@ -0,0 +1,2 @@
+datadog-checks-dev
+fiddler-client==1.4.0
```

Comment:
@rajeevthefiddler Maybe we should change this to include newer versions of the fiddler-client

```suggestion
fiddler-client>=1.4.0
```

Reply:
Done.